src/main.rs

```diff
@@ -59,51 +59,31 @@ fn main() {
         Ok(val) => Some(val),
         Err(why) => {
             // Error only if "no_panic_on_missing_launch_options" is not true.
-            if !(args.ignore_launch_options) {
-                panic!("Failed to get information on the current TF2 launch options from the local steam library: {}", why);
-            } else {
-                tracing::warn!("Couldn't verify app launch options, ignoring...");
-                None
-            }
+            tracing::warn!("Couldn't verify app launch options {:?}, ignoring...", why);
```

Comment:
Don't really need to specify that it's being ignored. Minor gripe though so I won't let it hold up the PR if you don't get around to changing it though.